sig-services/service-requirements/support-process.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Support process
+persona: service provider/owner, service consumer
+phase: service-development, service-design
+---
+
+An open service should have a support process defined. A support process is a way for users to reach out to the service owner/provider or developers and report any possible issues that might arise from the service e.g. service outage, feature request, or bug report. The support process should be transparent so that other users can see already solved issues and prevent duplicates and also see the progress of the reported issues. Response time to requests/reports shouldn't exceed the agreed time to respond and leave the user waiting for an answer, especially if the support process is also transparent. If the reported issue is a feature request or a bug the issue should not be closed before the change is available to the user in the next version.
+
+If a user needs to send any confidential information that is required for the issue, there should exist a way for the user to ensure confidentiality of the information, for example with some kind of encryption. The support process mustn't include a way to report vulnerabilities and security issues.
+
+
+## Requirements
+
+- Define a contact point for the support process (GitHub issues, email, ...)
```

Comment:
Should we recommend a place for defining the support process? maybe the user docs or readme of the Github service.

Reply:
That's a good idea I'll just add another sub-requirement that the support process should be documented in some place too.